src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sumA = 0;
```

Comment:
```suggestion
  let sumAccumulator = 0;
```
Let's use more meaningful names. `sumAccumulator` or `finalSum` would be more semantic